src/scripts/components/header/SearchInput.js

```diff
@@ -1,69 +1,85 @@
 import React from 'react';
-import {navigateAction} from 'fluxible-router';
+import {
+    navigateAction
+}
+from 'fluxible-router';
 import classNames from 'classnames';
 
 class SearchInput extends React.Component {
-  constructor(props, context) {
-    super(props, context);
-  }
+    constructor(props, context) {
+        super(props, context);
+    }
 
-  search(e) {
-    if (e.key === 'Enter' || e.keyCode === 13 || e.type === 'click') {
-      let searching = React.findDOMNode(this).querySelector('input').value,
-        ayah, pattern, surah;
+    search(e) {
+        if (e.key === 'Enter' || e.keyCode === 13 || e.type === 'click') {
+            let inputEl = React.findDOMNode(this).querySelector('input'),
+                searching = inputEl.value.trim(),
+                ayah, pattern, surah;
 
-      const shortcutSearch = /\d[\.,\:,\,,\\,//]/g;
-      const splitSearch = /[\.,\:,\,,\\,//]/g;
+            // prevent search function while search input field is empty
+            if (searching === '') {
```

Comment:
We use eslint for linting our JS which should have thrown an error or tests failing for this indentation (I will change it) but I am implementing 2 spaces indentation through-out. Following Airbnb's js: https://github.com/airbnb/javascript

I like 2 spaces more than 4 cause reactjs being its a blend of html & js could get really really messy really fast with 4 spaces and a pain to code review, hence 2 spaces :)